index.html

```diff
@@ -4,9 +4,58 @@
+            </div> 
+        <span class="orange-theme-color nav-text brand-name">dreamUI</span>
+        </div>
+        <div class="nav-bar-tabs mr-3">
+            <p class="nav-text nav-tab-text">Home</p>
+            <p class="nav-text nav-tab-text">Documentation</p>
+            <a href="https://github.com/dikshadhote/dreamUI/tree/dev"><i class="fab fa-github nav-text nav-tab-text" title="GitHub"></i></a>
+        </div>
+
+    </nav>
+    <div class="hero-img-container">
+        <img class="hero-img" src="./assets/undraw_mobile_development_re_wwsn.svg"/>
+        <div class="hero-text-container">
+            <h1 class="p-1 text-bold">Is building cool website your dream?</h1>
+            <p class="p-1 text-bold fs-2">Not anymore because <span class="orange-theme-dark-color brand-text">dreamUI</span> will make it a reality. </p>
+            <p class="p-1 text-bold fs-2">dreamUI will help you, to build website just as easy you take breath,</p>
+            <p class="p-1 text-bold fs-2">It will happen and you won't even realise...</p>
+            <button class="btn-primary">Get Started</button>
+        </div>
+    </div>
+    <footer class="footer">
+        <p class="white-text-color footer-text mt-3">Designed with love</p>
+        <small class="white-text-color">By</small>
+        <p class="white-text-color footer-text">Diksha Dhote</p>
+        <div class="footer-icons">
+            <a href="https://github.com/dikshadhote/dreamUI/tree/dev">
+                <i class="fab fa-github  white-text-color icon-footer p-1" title="GitHub"></i>
+            </a>
+            <a href="https://twitter.com/_Diksha_Dhote">
+                <i class="fab fa-twitter white-text-color icon-footer p-1" title="Twitter"></i>
+            </a> 
+            <a href="">
+                <i class="fab fa-linkedin white-text-color icon-footer p-1" title="Linkedin"></i>
+            </a> 
+        </div>
+        <p class="white-text-color footer-text">© dreamUI 2022, All rights reserved</p>
+        <a href="https://icons8.com/icon/7clwy8POIM6c/butterfly-side-view" class="white-text-color fs-small">Butterfly Side View icon by Icons8</a> 
+    </footer>
+    <script src="https://kit.fontawesome.com/7ec9fa6772.js" crossorigin="anonymous"></script>
 </body>
 </html>
```

Comment:
Add an extra new empty line at the end of your files
Refer to this for why you should: [https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log](url)

Reply:
> Add an extra new empty line at the end of your files Refer to this for why you should: [https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log](url)

added new line but didn't got clarity from this answer, will discuss with peers